raft/log_test.go

```diff
@@ -146,12 +146,14 @@ func TestAppend(t *testing.T) {
 
 // TestLogMaybeAppend ensures:
 // If the given (index, term) matches with the existing log:
-// 	1. If an existing entry conflicts with a new one (same index
-// 	but different terms), delete the existing entry and all that
-// 	follow it
-// 	2.Append any new entries not already in the log
+//  1. If an existing entry conflicts with a new one (same index
```

Comment:
This list should be rearranged I guess.

Reply:
Good spotting, the formatting tools were a bit annoying here but I think it's looking a bit better now.